axelor-helpdesk/src/main/resources/views/Charts.xml

```diff
@@ -11,22 +11,42 @@
 	  	<dataset type="sql">
 			SELECT 
 				COUNT(*) AS total_row, 
-				'ALL' as _status
+				(CASE WHEN(translation.message_value = '') THEN
```

Comment:
For null check, you can use CASE WHEN(COALESCE(translation.message_value,'')= '')
change it every where